src/components/content-box/_content-box.scss

```diff
@@ -3,24 +3,39 @@ $include-html: false !default;
 @if ($include-html) {
 
   .mint-content-box {
+
+    &__title {
```

Comment:
This container is supposed to have only headers. Some of them can wrap, like `mint-header--secondary`, hence the min-height.